zerver/tests/test_messages.py

```diff
@@ -1094,6 +1097,37 @@ def test_edit_message_no_content(self):
         content = Message.objects.filter(id=msg_id).values_list('content', flat = True)[0]
         self.assertEqual(content, "(deleted)")
 
+    def test_edit_message_history(self):
+        # type: () -> None
+        self.login("hamlet@zulip.com")
+        msg_id = self.send_message("hamlet@zulip.com", "Scotland", Recipient.STREAM,
+                                   subject="editing", content="content before edit")
+        new_content = 'content after edit'
+        params_dict = {'message_id': msg_id, 'content': new_content}
+
+        # Edits message content
+        result = self.client_patch("/json/messages/" + str(msg_id), params_dict)
+        self.assert_json_success(result)
+
+        request = self.Request()
+        request.GET = {} # Empty dict
+        request.POST = {'message_id': msg_id}
+
+        user = get_user_profile_by_email('hamlet@zulip.com')
+        message_edit_history = get_message_edit_history(request, user, message_id=msg_id)
```

Comment:
This should use the actual API endpoint (e.g. `self.client_get(/json/messages/...`) to get this history.  The response will be the same, but it will resolve the test failure complaining about the URL not being tested :).